OpenFusion.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
Is this from a fork? This shouldn't be pushed upstream. [2]

Reply:
fork